apps/meteor/ee/app/livechat-enterprise/server/api/monitors.ts

```diff
@@ -50,3 +59,44 @@ API.v1.addRoute(
 		},
 	},
 );
+
+const livechatMonitorsEndpoints = API.v1.post(
+	'livechat/monitors.save',
+	{
+		response: {
+			200: POSTLivechatMonitorsSaveSuccessResponse,
+			400: validateBadRequestErrorResponse,
+			401: validateUnauthorizedErrorResponse,
+			403: validateForbiddenErrorResponse,
+		},
+		authRequired: true,
+		permissionsRequired: ['manage-livechat-monitors'],
+		license: ['livechat-enterprise'],
+		body: isPOSTLivechatMonitorSaveRequest,
+	},
+	async function action() {
+		const { username } = this.bodyParams;
+
+		try {
+			const result = await LivechatEnterprise.addMonitor(username);
+			if (!result) {
+				return API.v1.failure('error-adding-monitor');
```

Comment:
Could add a i18n translation for this error code? 
It would improve UX by displaying a readable error message in the UI in case of errors.